pkg/sql/logictest/testdata/logic_test/hash_sharded_index

```diff
@@ -276,26 +276,26 @@ CREATE TABLE sharded_secondary (a INT8, INDEX (a) USING HASH WITH BUCKET_COUNT=1
 query TTTTT
 EXPLAIN (VERBOSE) INSERT INTO sharded_secondary (a) VALUES (1), (2)
 ----
-·                           distributed    false                                              ·                                    ·
-·                           vectorized     false                                              ·                                    ·
-count                       ·              ·                                                  ()                                   ·
-    └── insert                 ·              ·                                                  ()                                   ·
-        │                     into           sharded_secondary(a, crdb_internal_a_shard_12, rowid)   ·                                    ·
-        │                     strategy       inserter                                           ·                                    ·
-        │                     auto commit    ·                                                  ·                                    ·
-        └── render            ·              ·                                                  (column1, column6, column5, check1)  ·
-            │                render 0       column1                                            ·                                    ·
-            │                render 1       column6                                            ·                                    ·
-            │                render 2       column5                                            ·                                    ·
-            │                render 3       column6 IN (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11)  ·                                    ·
-            └── render       ·              ·                                                  (column6, column5, column1)          ·
-                │           render 0       mod(fnv32(column1::STRING), 12)                    ·                                    ·
-                │           render 1       unique_rowid()                                     ·                                    ·
-                │           render 2       column1                                            ·                                    ·
-                └── values  ·              ·                                                  (column1)                            ·
-·                           size           1 column, 2 rows                                   ·                                    ·
-·                           row 0, expr 0  1                                                  ·                                    ·
-·                           row 1, expr 0  2                                                  ·                                    ·
+·                           distributed    false                                                  ·                                    ·
```

Comment:
While you're here, can you move this from logictest into the execbuilder tests? Tests that look at plans shouldn't be in the logictest/ dir.